pkg/loop/internal/pb/contract_reader.proto

```diff
@@ -20,6 +21,7 @@ message GetLatestValueRequest {
   string read_identifier = 1;
   Confidence confidence = 2;
   VersionedBytes params = 3;
+  bool as_value_type = 4;
```

Comment:
VersionedBytes should take care of this, just return the same version the user sends you.